clusterman/cli/toggle.py

```diff
@@ -55,6 +55,10 @@ def disable(args: argparse.Namespace) -> None:
         "timestamp": {"N": str(int(time.time()))},
     }
 
+    if not args.until:
+        print("Default has changed; autoscaler will be re-enabled in 30m since no --untill flag was used.")
```

Comment:
should we set this default closer to where the arg is defined and unconditionally print how long the autoscaler will be paused for?

Reply:
and if we decide to keep this:
```suggestion
        print("Default has changed; autoscaler will be re-enabled in 30m since no --until flag was used.")
```

Reply:
Initially I just changed Line 108 to add the default there, but that won't communicate the changed default to users, so I prefer this way, at least for a couple versions and until people have noticed the changed default.